hikari/impl/shard.py

```diff
@@ -342,7 +342,7 @@ class GatewayShardImpl(shard.GatewayShard):
     ----------------
     compression : typing.Optional[buitlins.str]
         Compression format to use for the shard. Only supported values are
-        `"payload_zlib_stream"` or `builtins.None` to disable it.
+        `"transport_zlib_stream"` or `builtins.None` to disable it.
```

Comment:
Actually, this is wrong, we only allow `"zlib-stream"`

Reply:
That's the raw value we send to discord, not the value of TRANSPORT_ZLIB_STREAM

Reply:
True